doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/index.md

```diff
@@ -2,304 +2,198 @@
-
-<!-- TODO: Fix this link when Administration-UI-guide#AdministrationUIguide-GeneratingSDK are updated -->
-To build the Java endpoint SDK, [generate](Administration-UI-guide#AdministrationUIguide-GeneratingSDK) the Java endpoint SDK in Admin UI and download the generated .jar file.
-
-# C endpoint SDK
-
-## Install build utils and dependencies
+To build C SDK make sure that the following components are installed:
 
-Before building the C endpoint SDK, install the following components on your machine:
+ - CMake (minimum required version is 2.8.8)
+ - C99 compatiable compiler (e.g. GCC)
+ - OpenSSL
 
-1. Install compilers:
-    1. For automatic installation, execute the following commands:
+To install dependencies on Ubuntu proceed as follows:
 
-       ~~~ shel
-       $ sudo apt-get install gcc
-       ~~~
+```
+sudo apt-get install cmake libssl-dev gcc
+```
 
-    2. For manual installation of version 4.8, refer to the following example:
+### Build procedure
 
-       ~~~ shell
-       $ sudo apt-get install python-software-properties
-       $ sudo add-apt-repository ppa:ubuntu-toolchain-r/test
-       $ sudo apt-get update
-       $ sudo apt-get install gcc-4.8
-       $ sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-4.8 50
-       ~~~
-
-2. Install CMake utility:
-    1. For automatic installation, execute the following commands (tested on Ubuntu 14.04):
-
-       ~~~ shell
-       $ sudo apt-get install cmake
-       ~~~
-
-    2. For manual installation, refer to the following example:
-
-       ~~~ shell
-       $ wget http://www.cmake.org/files/v3.3/cmake-3.3.0-rc2.tar.gz
-       $ tar -zxf cmake-3.3.0-rc2.tar.gz
-       $ cd cmake-3.3.0-rc2/
-       $ ./configure
-       $ sudo make install
-       ~~~
-
-3. Install OpenSSL:
+**Verified against:**
 
-   ~~~
-   $ sudo apt-get install libssl-dev
-   ~~~
+**Host OS:** Ubuntu 14.04 LTS Desktop 64-bit.
 
-4. Install CUnit:
+After you generated archive with C SDK, proceed as follows:
 
-   ~~~
-   $ sudo apt-get install libcunit1-dev
-   ~~~
+1. Unpack sdk:
 
-## Compile C endpoint SDK
+        tar -xvf c-sdk-archive-name.tar.gz
 
-To build the C endpoint SDK, do the following:
+2. Create the directory where sdk will be built:
 
-<!-- TODO: Fix this link when Administration-UI-guide#AdministrationUIguide-GeneratingSDK are updated -->
-1. [Generate ](Administration-UI-guide#AdministrationUIguide-GeneratingSDK)the C endpoint SDK in Admin UI.
-2. Download and untar the Kaa C SDK archive.
 
-   ~~~ bash
-   $ tar xfv kaa-c-ep-sdk.tar.gz
-   ~~~
+        mkdir build
+        cd build
 
-3. Run the following commands.
+3. Configure the build via cmake:
 
-   ~~~ bash
-   $ mkdir build
-   $ cd build
-   $ cmake ..
-   $ make
-   $ make install
-   ~~~
+        cmake ..
 
-## Configure C endpoint SDK
+4. Perform build:
 
-To configure the C endpoint SDK build, you can optionally specify the following parameters for the **cmake** command.
+        make
 
+## Installation
```

Comment:
I propose to drop it. C SDK is auto-generated and not versioned, thus likely that no one will ever install it.